hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -257,7 +257,17 @@ public final class OzoneConfigKeys {
   public static final String OZONE_BLOCK_DELETING_SERVICE_TIMEOUT =
       "ozone.block.deleting.service.timeout";
   public static final String OZONE_BLOCK_DELETING_SERVICE_TIMEOUT_DEFAULT
-      = "300s"; // 300s for default
+      = "300s"; // 300s for default deleteBlocksCommandLimit
+
+  /**
+   * A limit to restrict the total number of delete block commands queued on a
+   * datanode. Note this is intended to be a temporary config until we have a
+   * more dynamic way of limiting load.
+   */
+  public static final String OZONE_BLOCK_DELETING_PENDING_COMMAND_LIMIT =
+      "ozone.block.deleting.pending.command.limit";
+  public static final int OZONE_BLOCK_DELETING_PENDING_COMMAND_LIMIT_DEFAULT
+      = 5; // same with hdds.datanode.block.delete.queue.limit
```

Comment:
Can we reuse the existing `hdds.datanode.block.delete.queue.limit` configuration instead of adding a new, temporary one?

Reply:
If the configuration is applicable to both DN and SCM, they can share the same config.  I think that's better than having two configs and instructing admins to ensure they have the same value.

For example we use Datanode's `hdds.datanode.replication.streams.limit` configuration similarly in SCM's `ReplicationManager`, too.

Reply:
done.

Reply:
Thanks.  Test needs to be updated as well:

https://github.com/xichen01/ozone/actions/runs/5402403295/jobs/9813642595#step:6:1600